packages/blockchain-wallet-v4-frontend/src/modals/BuySell/SellEnterAmount/Checkout/Payment/Payment.styles.ts

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components'
+
+export const DisplayIconAligned = styled.div`
```

Comment:
There's a `<Flex />` component in constellation that could replace this.

Reply:
This file is just a copy of what we already had. Definitely something to refactor 👍